readme.md

```diff
@@ -19,5 +19,5 @@ Please use `grid` this time around.
 
 **Result**
 
-- [UPDATE DEMO LINK](https://<your_account>.github.io/<repo_name>/)
-- [UPDATE TEST REPORT LINK](https://<your_account>.github.io/<repo_name>/report/html_report/)
+- [UPDATE DEMO LINK](https://KonstVolkovnitsky.github.io/layout_antisnake/)
+- [UPDATE TEST REPORT LINK](https://KonstVolkovnitsky.github.io/layout_antisnake/report/html_report/)
```

Comment:
the solution did not pass all the tests.
Try change solution and pass all tests

Reply:
@gestap could you please help @KonstVolkovnitsky 
Because I could not find the reason for failing tests in his code